mne/event.py

```diff
@@ -259,10 +259,16 @@ def read_events(filename, include=None, exclude=None, mask=0):
         event_list = lines[:, goods]
         if event_list.shape[0] > 0 and event_list[0, 2] == 0:
             event_list = event_list[1:]
+            warn('first row of event file discarded (zero-valued)')
```

Comment:
Also, I would like this discarding of the first row to not happen if `mask=None` but I don't know if there is a use case where discarding the first row is desired but binary masking is not.  Any guidance?